### DISCUSSION TITLE
No IntelliSense for string with ?: and string()

### DISCUSSION THREAD

**Discussion by tasty0tomato** [microsoft/vscode-cpptools#10060] [Q&A]
A small test
<img width="165" alt="Code_GfRtSIDspY" src="https://user-images.githubusercontent.com/48764449/197775739-7b03956f-17c4-46c1-aec3-c8a3c30a49ae.png">
If not string(), there will be IntelliSense 
<img width="263" alt="Code_kA2BlXYba8" src="https://user-images.githubusercontent.com/48764449/197775802-2726602a-a2dc-4f3e-9ced-44c5ad18cb8f.png">
So maybe not just a problem about ?:

**Comment by sean-mcmanus** [comment]
@tasty0tomato It's a bug. I've moved it to https://github.com/microsoft/vscode-cpptools/issues/10103

**Reply by sean-mcmanus** [reply]
The fix is available with 1.14.0 (pre-release): https://github.com/microsoft/vscode-cpptools/releases/tag/v1.14.0
